kola/tests/ignition/units.go

```diff
@@ -0,0 +1,56 @@
+// Copyright The Mantle Authors
+// Copyright 2020 Red Hat
+// SPDX-License-Identifier: Apache-2.0
+package ignition
+
+import (
+	"github.com/flatcar-linux/mantle/kola/cluster"
+	"github.com/flatcar-linux/mantle/kola/register"
+	"github.com/flatcar-linux/mantle/platform/conf"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Name:        "cl.ignition.instantiated.enable-unit",
+		Run:         enableSystemdInstantiatedService,
+		ClusterSize: 1,
+		UserData: conf.Ignition(`{
+    "ignition": {"version": "3.0.0"},
+    "systemd": {
+        "units": [{
+			"name": "echo@.service",
+			"contents": "[Unit]\nDescription=f\n[Service]\nType=oneshot\nExecStart=/bin/echo %i\nRemainAfterExit=yes\n[Install]\nWantedBy=multi-user.target\n"
+		  },
+		  {
+		  "name": "echo@.timer",
+		  "contents": "[Unit]\nDescription=echo timer template\n[Timer]\nOnUnitInactiveSec=10s\n[Install]\nWantedBy=timers.target"
+		},
+		{
+		  "enabled": true,
+		  "name": "echo@bar.service"
+		},
+		{
+		  "enabled": true,
+		  "name": "echo@foo.service"
+		},
+		{
+			"enabled": true,
+			"name": "echo@foo.timer"
+		}]
+    }
+}`),
+		Distros: []string{"cl"},
+	})
```

Comment:
@tormath1 this need a MinVersion doesn't it?

Reply:
Oh yes, version 3.0.0... Totally missed this one.